src/Native/Runtime/thread.h

```diff
@@ -135,7 +135,7 @@ class Thread : private ThreadBuffer
     void ClearState(ThreadStateFlags flags);
     bool IsStateSet(ThreadStateFlags flags);
 
-    static UInt32_BOOL HijackCallback(HANDLE hThread, PAL_LIMITED_CONTEXT* pThreadContext, void* pCallbackContext);
+    static UInt32_BOOL _stdcall HijackCallback(HANDLE hThread, PAL_LIMITED_CONTEXT* pThreadContext, void* pCallbackContext);
```

Comment:
I would rather remove `__stdcall` from both here and the header. Also, could you please fix the return type to UInt32 while you are on it to match the header?